code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -47,3 +55,125 @@
 	if(istype(weapon, /obj/item/melee/flyswatter))
 		return 9 //flyswatters deal 10x damage to moths
 	return 0
+
+/datum/species/moth/on_species_gain(mob/living/carbon/human/H)
+	..()
+	H.dna.original_wings = "[H.dna.features["moth_wings"]]" //This stores the current wing type
```

Comment:
You should probably just add a seperate "burnt" var on moths that overrides the wings texture, instead of changing their DNA.

Reply:
That's kind of how other servers handle it, they apply a debuff which changes how the wings look. How they do it though I couldn't understand.

Reply:
you just put a burnt var on the moth species datum and then set that to true, then wherever determines how the wings look you check if burnt is true and display the burnt wings instead of the normal ones.